<launch>
	<!-- <include file="$(find ias_gscam)/gscam_640.launch"/> -->

	<node name="tracker" pkg="aruco_pose" type="tracker" output="screen">
		<remap from="image" to="/gscam/image_raw"/>
		<param name="correction_table_path" value="$(find aruco_pose)/bin/MicroEdgeCorrectionTable.bin" type="string"/>
		<param name="marker_size" value="1.0" type="double"/>
		<param name="publish_markers" value="true" type="bool"/>
		<param name="publish_transforms" value="true" type="bool"/>
	</node>
</launch>
